### ISSUE TITLE
Missing flag

### ISSUE THREAD

**Issue by Giolly90** [barbeau/gpstest#247]
**Summary:** 

Flag for satellite 331 is missing, please also change the flag for all Galileo satellites with a standard EU flag, not the Galielo one it will be consistent with the rest (plus flags are too small and not easy to understand what is shown)

**Expected behavior:** 

Show EU flag

**Device and Android version:** 

OnePlus5 OxygenOS Android Pie 9.0.0

**Screenshot:**

![screenshot_20190109-001949](https://user-images.githubusercontent.com/46520205/50891314-49a8db00-13fc-11e9-8868-57039f4477a9.jpg)

**Comment by barbeau** [comment]
@Giolly90 Interesting! I can't find a reference to 331 anywhere online. 

Is this a stock install of Oxygen OS on your OnePlus5? And what country are you in?

A missing GNSS flag on Android 9 stock install means that the GNSS isn't recognized by the Android OS. Top contender would be IRNSS as it doesn't yet have an Android system constant.

Anyone else have a guess for what this is?

**Comment by Giolly90** [comment]
Yes, I'm on stock OxygenOS and the country is Italy.
In my opinion satellite 331 is Galileo FOC FM18, isn't it?

**Comment by barbeau** [comment]
@Giolly90 I'm not sure - do you always see this satellite without a flag, or does it only occasionally appear?

Either way, if it is Galileo and you're on stock Android OS, this is a bug in your device and you should report it to OnePlus.  Their firmware isn't reporting the GNSS constellation correctly to Android for that satellite.

**Comment by Giolly90** [comment]
I saw 331 only once, I'll give you update if I see it again.

What about the EU flag request? Can you change with this?

**Comment by barbeau** [comment]
@Giolly90 I'm considering changing the Galileo satellites flag to the EU flag...But no decision yet.

**Comment by Giolly90** [comment]
The Galileo one is a symbol, like these two, the flag is another story
![beidou_logo](https://user-images.githubusercontent.com/46520205/51253689-564ea580-199f-11e9-8dc5-a030b79fb6a7.png)
![glonass_logo](https://user-images.githubusercontent.com/46520205/51253691-564ea580-199f-11e9-948b-582ec8dfbec7.png)

**Comment by Giolly90** [comment]
331 spotted again!

**Comment by Giolly90** [comment]
Keep spotting 331
